base/services/tftpd/tftpd.cpp

```diff
@@ -540,11 +534,7 @@ void processRequest(void *lpParam)
                 MYDWORD iip = ntohl(req.client.sin_addr.s_addr);
                 bool allowed = false;
 
-#ifdef __REACTOS__
```

Comment:
Are `#ifdef __REACTOS__` block removals intended?

Reply:
Do you really think it's necessary?

Reply:
I don't know: that is what I am asking you, since you did not mention them explicitly.